HaxeManual/08-compiler-features.tex

```diff
@@ -734,3 +734,28 @@ \section{Static Analyzer}
 \item[\ic{fusion}]: Moves variable expressions to its usage in case of single-occurrence. By default, only compiler-generated variables are handled. This can be changed by using the compiler flag `\ic{-D analyzer-user-var-fusion} or the metadata \ic{@:analyzer(user_var_fusion)}.
 \item[\ic{purity_inference}]: Infers if fields are ``pure'', i.e. do not have any side-effects. This can improve the effect of the fusion module.
 \end{description}
+
+
+\section{Loop unrolling}
+\label{cr-loop-unrolling}
+\since{4.0.0}
+
+The compiler tries to unroll short iterations of \expr{for} loops over constant ranges. This can produce faster code that uses fewer variables.
```

Comment:
Not sure if "fewer variables" is the significant thing to mention here. The main point is that there's no longer a loop at all.